Add post for 0.14.2 release (#101)
TeamNewPipe/website: _posts/2018-10-16-newpipe-0.14.2-released.md

---
layout: post
title: NewPipe 0.14.2 released, introducing grid layout for tablets
short: "NewPipe 0.14.2 released"
date: 2018-10-16 13:00:00 +0200
categories: [release, pinned]
author: TobiGr
image: newpipe
excerpt_separator: <!-- more -->
---

> We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements

Reviewer: "Such as" is the wrong wording here.

```suggestion
We have just released NewPipe 0.14.2. Just like NewPipe 0.14.0 this version mostly brings UI and UX improvements
```

> which include new layouts for tablets and an additional linear layout in "Downloads".

Reviewer: ```suggestion
including new layouts for tablets and an additional linear layout in "Downloads".
```

> Furthermore, the subscriptions page got a small upgrade, several bugs were fixed and new translations added.

Reviewer: Try to avoid "get" if possible. I'd further split the sentence into two parts.

```suggestion
Furthermore, the subscriptions page receievd a small upgrade. Also, several bugs were fixed and new translations were added.
```

<!-- more -->

## New tablet UI

> [@nv95](https://github.com/nv95) brings a completely new feeling to NewPipe on tablets: he implemented grid layouts for video lists like the search, a channel's videos or the trending page. He took a look at the subscriptions page, too:

Reviewer: ```suggestion
[@nv95](https://github.com/nv95) brings a completely new feeling to NewPipe on tablets. He implemented grid layouts for video lists like for example the search, a channel's videos or the trending page. He took a look at the subscriptions page, too:
```

<img class="no-flow vertical" src="/img/screenshots/shot_subscriptions_tablet_dark.png"/>

> On top of that, he also made the video detail view responsive. This means, next and recommended videos will be displayed at the right side of your screen if your device is large enough.

Reviewer: No comma here.

```suggestion
On top of that, he also made the video detail view responsive. From now on, next and recommended videos will be displayed at the right side of your screen if your screen's resolution is large enough.
```

<img class="no-flow vertical" src="/img/screenshots/shot_video_detail_tablet_dark.png"/>

[@jludden](https://github.com/jludden) implemented an additional list layout for the downloads. This might help with these videos which titles have the same beginnings.
<!-- TODO: linear layout download image -->
<img class="no-flow" src="/img/screenshots/shot_05.png"/>

## Improved subscription handling

> This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This is helpful whenever a channel was deleted or renamed by YouTube. So if you have had dead channels before, it is now time to clean them up!

Reviewer: ```suggestion
This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This can be really useful whenever a channel was deleted or renamed by YouTube. So, if you have had dead channels before, it is now time to clean them up!
```

And for those, who have discovered incorrect subscription counts in the past, this will not happen again - hopefully.

## Further changes

> [@ritiek](https://github.com/ritiek) added separate gesture controls to the player. This means, you can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.

Reviewer: Avoid "This means", it's redundant in this context.
```suggestion
[@ritiek](https://github.com/ritiek) added separate gesture controls to the player. You can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.
```

> [@MaX-Lo](https://github.com/MaX-Lo) made the player remember the last aspect ratio. In case you have a 18:9 device or just like to view video in another than the default aspect ratio, you will definitely enjoy this improvement. 

Reviewer: Is 18:9 really a thing?

Author: Yes. See https://github.com/TeamNewPipe/NewPipe/issues/1318#issuecomment-382398636.
You can find several devices using a search engine of your choice :P
Here is just a small collection: Samsung Galaxy S8, LG G6, Huawei Nova 2 / P10, Sony Xperia XZ1 Compact. Even Apple use this ratio for the IPhone 8 and SE.

This is a significant enhancement for 2:1 devices as well. So if you are more comfortable with 2:1, we can use this instead.

Reviewer: I wonder why they call it 18:9 instead of 2:1... Thanks for the heads-up. You should add a link to that issue!

Author: If I add a link to this issue, I should add links to most other addressed issues, too. Just for consistency. I guess there will be some more changes before we release the new version. With other words, this can wait till tomorrow

[@coin3x](https://github.com/coin3x) fixed the video queuing behaviour. When the latest queue already finished playing, enqueuing a new video or audio will start the stream by its own.

> [@theScrabi](https://github.com/theScrabi) worked on improving NewPipe's localization handling. For devices where the content country and the actual location differ, the number of views and video descriptions are now displayed in the correct language. 

Author: @theScrabi I am pretty sure this paragraph contains false information about your localization changes. What exactly did you fix in https://github.com/TeamNewPipe/NewPipe/pull/1792?

Reviewer: This should indeed be fact checked.

He also added a new content setting which allows you to set the content language. 

> Speaking of localization, we would like to thank all translators who updated NewPipe and added support for two new languages: (Don't remember which, will insert here once Weblate was merged. Taking a look into the commit summary is less time expensive then scrolling through the history) All together, NewPipe comes with translations for nearly [55 languages](https://hosted.weblate.org/projects/newpipe/#languages)! If you find incomplete or missing translations in your language, feel free to [improve them](https://hosted.weblate.org/engage/NewPipe/).

Reviewer: Big, fat ***FIXME*** in that line. Please fix before merge.

Reviewer: "All together" should be "Altogether", shouldn't it?

Author: > Big, fat FIXME in that line. Please fix before merge.

I know that this information misses. That's why I added the ToDo list above. I need to wait for @theScrabi to merge WebLate.

>"All together" should be "Altogether", shouldn't it?

yes.

[@BO41](https://github.com/BO41) cleaned up the code which results in a smaller app size and tiny performance improvements.

> [@theScrabi](https://github.com/theScrabi) took a look at another bug in this version: after fixing time parsing for formats like "02.56", people can use NewPipe in Finland now.

Reviewer: I'd add "... and other countries with strange date formatting". Show the Finnish how weird their language how weird their language is!

Author: Hm. I wouldn't do that :D
